src/Microsoft.AspNet.Mvc.TagHelpers/js/LinkTagHelper_FallbackJavaScript.js

```diff
@@ -0,0 +1,11 @@
+﻿(function (cssTestPropertyName, cssTestPropertyValue, fallbackHref) {
+    // This function finds the previous element (assumed to be meta) and tests it current CSS style using the passed
+    // values, to determine if a stylesheet was loaded. If not, it loads the fallback stylesheet via document.write.
+    var doc = document,
+        scriptElements = doc.getElementsByTagName("SCRIPT"),
+        meta = scriptElements[scriptElements.length - 1].previousElementSibling;
```

Comment:
isn't this fragile?  how can we be sure the page won't include multiple `<script/>` elements after the `<meta/>` tag of interest?